zingocli/Cargo.toml

```diff
@@ -28,27 +28,6 @@ zingo-testutils = { path = "../zingo-testutils" }
 portpicker = { workspace = true}
 json = "0.12.4"
 
-[dev-dependencies]
-zcash_primitives = { workspace = true }
-tempdir = { workspace = true }
-portpicker = { workspace = true }
-env_logger = "0.10.0"
-zingolib = { path = "../zingolib/" }
-bech32 = "0.9.0"
-rand = "0.8.5"
-hex = "0.3"
-tracing-subscriber = "0.3.15"
-itertools = "0.10.5"
-bip0039 = "0.10.1"
-tracing-test = { version = "0.2.4", features = ["no-env-filter"] }
-tracing = "0.1.37"
-tracing-log = "0.1.3"
-futures-util = "0.3.28"
-orchard = { workspace = true }
-zcash_address = { workspace = true }
-zcash_client_backend = { workspace = true }
-shardtree = { workspace = true }
-serde_json = "1.0.107"
 
 [features]
 default = ["local_env"]
```

Comment:
if you have removed local env is this needed anymore?

Reply:
Fixed.